test-requirements.txt

```diff
@@ -5,3 +5,6 @@ charms.reactive
 mock>=1.2
 coverage>=3.6
 git+https://github.com/openstack/charms.openstack.git#egg=charms-openstack
+charmhelpers
```

Comment:
I believe we should be using `git+https://github.com/juju/charm-helpers.git#egg=charmhelpers` would be better for once this charm gets released we can switch to the `stable/*` branch